prov/verbs/src/ep_rdm/verbs_queuing.h

```diff
@@ -114,20 +114,23 @@ fi_ibv_rdm_tagged_move_to_postponed_queue(
 {
 	FI_IBV_RDM_TAGGED_DBG_REQUEST("move_to_postponed_queue: ", request,
 				      FI_LOG_DEBUG);
-	assert(request->conn);
+	assert(request && request->conn);
 
-	if (dlist_empty(&request->conn->postponed_requests_head)) {
+	struct fi_ibv_rdm_tagged_conn *conn = request->conn;
+
+	if (dlist_empty(&conn->postponed_requests_head)) {
 		struct fi_ibv_rdm_tagged_postponed_entry *entry =
 			(struct fi_ibv_rdm_tagged_postponed_entry *)
-			fi_verbs_mem_pool_get(&fi_ibv_rdm_tagged_postponed_pool);
+			fi_ibv_mem_pool_get(&fi_ibv_rdm_tagged_postponed_pool);
```

Comment:
As a separate patch, can we please replace the verbs memory pool with the util_pool that is now available?  See include/fi_mem.h